kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -270,23 +279,31 @@ func NewDispatcher(args *KafkaDispatcherArgs) (*KafkaDispatcher, error) {
 		return nil, fmt.Errorf("unable to create kafka producer: %v", err)
 	}
 
+	ceClient, err := kncloudevents.NewDefaultClient()
```

Comment:
not needed for this PR, but I think it would be worth to look at the good_client in eventing, and see if we can use that one, or update the one in eventing_contrib... https://github.com/knative/eventing/blob/master/pkg/kncloudevents/good_client.go
I added some tuning there, where you can better reuse connections, and that improved perf quite a bit... 
something to keep in mind...

Reply:
https://github.com/knative/eventing-contrib/issues/796 created an issue to track this in it's own PR